env0/data_azure_credentials.go

```diff
@@ -0,0 +1,93 @@
+package env0
+
+import (
```

Comment:
Why do we need a `data` source for this? I see we have it for the other credentials, but it seems very unnecessary to me (and it won't even work as far as I understand, because we redact the secret values)

Reply:
[0f20b41](https://github.com/env0/terraform-provider-env0/pull/258/commits/0f20b41f2f2eeeee4eee08cad889f47c3bdb3824)
[d46d1de](https://github.com/env0/terraform-provider-env0/pull/258/commits/d46d1ded48b6adffddb842b135e81fcae9a29d2c)

Reply:
I think is very important data here.

1. manage the credential from the UI and only connect it to the project.
2. manage the credential in other stuck (organization stack)

Reply:
I meant you manage X stacks for our provider for example:

org stack:
```
resource azure_cred {
   name: 'my_production_keys'
}
```

project stack:
```
data azure_cred {
  name: 'my_production_keys'
}

resource env0_project_credentials  {
  project_id: env0.project.id
  cred_id: data.azure_cred.id
  type: AzureSecret
}
```